rootstrap/rails_hotwire_base
# Add Flipper for feature flags

**Description:**
This PR integrates the [Flipper gem](https://github.com/jnunemaker/flipper) with the `ActiveSupportCacheStore` adapter working on top of the `ActiveRecord` adapter, so that we don't have to go to the DB to know the value of the feature flags every time.

**QA Instructions:**
To see that this is working I added a `test` feature flag that displays a text in the root path if it's enabled.

1. Go to the admin panel and log in
2. Click on the `Feature Flags` link in the sidebar menu
3. Fully enable the `test` feature flag
4. Log out of the admin panel
5. Go to the root path and check that the text is now being displayed
6. You can go back to the admin to disable the feature flag and see that the text isn't shown anymore

**Notes:**
The code added to test that this is working will be removed once it is approved to be merged.
